css/base.css

```diff
@@ -0,0 +1,18 @@
+li {
+  list-style: none;
+  padding-left: 1em;
+}
+body {
+  background-color: #F7FFEF;
+  font-family: Architects Daughter, cursive;
+}
+img {
+  width: 2em;
+  max-width:100%;
+
```

Comment:
@laceylin2010 minor nitpick - got a few extra spaces here - best practice is to keep all of your encapsulated css together and each css rule should have a space between them...  example:

div {
  color: green;
  background: orange;
  border: solid 1px orange;
}

p {
  color: blue;
  text-align: center;
}